### ISSUE TITLE
FR: starlark script does not work on metrics

### ISSUE THREAD

**Issue by dpajin-kentik** [kentik/ktranslate#481]
When I associate a starlark script with the metric of the table, I don't see any actions performed by script. 

I associate it in the following way
```

      - tag: cefcModuleAdminStatus
        OID: .1.3.6.1.4.1.9.9.117.1.2.1.1.1
        name: cefcModuleAdminStatus
        enum:
          enabled: 1
          disabled: 2
          reset: 3
          outOfServiceAdmin: 4
          unsupported: 0
        script: |
          def main(n):
            print("cefcModuleAdminStatus")
            print(n)
            return None
```
I don't see anything printed or executed. although in logs it says that it is enabling script:

```
2022-11-21T16:06:21.219 ktranslate/ [Info] KTranslate Enabling script for cefcModuleAdminStatus
2022-11-21T16:06:21.219 ktranslate/ [Info] KTranslate>cat6509 Adding device metric .1.3.6.1.4.1.9.9.117.1.2.1.1.1 -> cefcModuleAdminStatus
```

Script works so far with the metric_tags, but not with the metric. 

- I want to be able to perform operations on both metrics and metric tags from the same table
- I want to be able to add additional metrics and metric tags in the table, using script
- I want to be able to remove metric or metric tags from the results using script

**Comment by dpajin-kentik** [comment]
- [ ]  I want to be able to perform operations on both metrics and metric tags from the same table
    - This should be configurable by associating script at the table level. Preferably, associating script file. Something like

```
  - MIB: ABC-MIB
    table:
      OID: .1.3.6.1.2.1.XX   
      name: nameOfTheTable
      script: |...
or
      script_file: script_for_changing_stuff.star

```

- [ ]  I want to be able to add additional metrics and metric tags in the table, using script
    - This should be possible by properly exposing `event` to the starlark script. If the script starts with `def main(event):` The event should contain:
        - event.timestamp - timestamp
        - event.tags - dictionary of tags
        - event.metrics - dictionary of metrics
        - event.name - this is probably not needed or not possible to implement, as the influxdb measurement name is constructed from “MIB” and “table” names.
    - To add some tags/metrics or delete some, this should be performed over the object handled to the script.
    - The script should return modified event object back to ktranslate by doing `return event`
- [ ]  I want to be able to remove metric or metric tags from the results using script
    - If the script returns `return None` that means that the event should be filtered out and not sent further.

**Comment by i3149** [comment]
Up to you I guess but consider closing this because #492 was merged.

**Comment by dpajin-kentik** [comment]
I tested #492 and it works, but I still have a problem as this doesn't work exactly how I would like. At the moment, script on tag value is able to see all the tags and script on the metric is able to see all the metrics. But it is not possible to see/change metrics while being able to see tags, and vice versa.

I think that the behavior which I described above would be much more convenient? is there any limitation specific limitation which make that hard to achieve?
